raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -348,6 +349,7 @@ def next_transfer_pair(
         payer_transfer: LockedTransferSignedState,
         available_routes: List['RouteState'],
         channelidentifiers_to_channels: Dict,
+        pseudo_random_generator,
```

Comment:
Same as above

Reply:
I did not add the type annotation for functions that are missing it for most arguments, the goal of this PR is not to type annotate the module. For the other modules which have more annotations (initiator and initiator_manager) I added the annotations to most functions.